<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab2 client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="lwLog"/>
   </item>
   <item row="2" column="0">
    <widget class="QPlainTextEdit" name="pteMessage"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ADC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLineEdit" name="leHost">
       <property name="text">
        <string>169.254.153.204</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPort">
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDisconnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widget1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>185</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>frequence</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OUT_DOUBLE_ADC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UART ENABLE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double ADC 1kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double ADC 10kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double ADC 50kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EXT ADC1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADC1 1kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADC1 10kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADC1 50kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SINGLE ADC2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>Choose Periphery</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ethernet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UART</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSend">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpenFile">
         <property name="text">
          <string>Open File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
